packages/block-library/src/template-part/edit/selection/template-part-previews.js

```diff
@@ -109,7 +109,8 @@ function TemplatePartsByTheme( {
 	return templatePartsByTheme.map( ( templatePartList ) => (
 		<PanelGroup
 			key={ templatePartList[ 0 ].meta.theme }
```

Comment:
I think we should add a fallback value here as well, to avoid an empty key. 🤔

Reply:
Ah I see I was late here 😄

Reply:
Possibly?  I think React doesn't get mad if the key is the empty string, like it does when it is `null`/`undefined`.  So in this case `''` may still be a valid unique key for the group?

Reply:
Yeah I've noticed React didn't complain about it, so... LGTM then!